canonical/snapcraft.io
# feat: Move beta store to store page

## Done
Moved the beta store to the store page

## How to QA
- Go to https://snapcraft-io-4621.demos.haus/store
- Check that if a category is selected that it is shown as the title
- Check that if there are two categories selected that the title is CATEGORY_NAME and 1 more category
- Check that if there are more than two categories selected that the title is CATEGORY_NAME and X more categories
- **Note**: The beta JS will be renamed to store next week but that is too risky a change for a Friday at a sprint

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-3219
